lua/nvim-treesitter/parsers.lua

```diff
@@ -1763,6 +1763,15 @@ return {
     maintainers = { '@stsewd', '@theHamsta' },
     tier = 2,
   },
+  qf = {
+    install_info = {
+      revision = 'v1.2.0',
+      url = 'https://github.com/OXY2DEV/tree-sitter-qf',
+    },
+    maintainers = { '@OXY2DEV' },
+    tier = 1,
```

Comment:
I think it's better to keep this at tier 2 for now, if the parser isn't stable yet.